README.md

```diff
@@ -1,70 +1,165 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-In the project directory, you can run:
+  <h3><b>Microverse README Template</b></h3>
 
-### `npm start`
+</div>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📗 Table of Contents
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[your\_project\_name\] ](#-your_project_name-)
+  - [🛠 Built With React Js](#-built-with-react-js)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Deployment ](#-live-deployment-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+- [Deployment](#deployment)
+- [type `npm start`](#type-npm-start)
+    - [Prerequisites](#prerequisites-1)
+    - [Run tests](#run-tests)
+- [type `npm test`](#type-npm-test)
+    - [Deployment](#deployment-1)
+- [type `npm run build`](#type-npm-run-build)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
 
-### `npm test`
+# 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- Optional
Consider giving it a proper title, the name of the project would sound much better. This is one of those projects you will put on your portfolio. It is important have a professional README file.